### DISCUSSION TITLE
Log endpoint detection seems to be missing

### DISCUSSION THREAD

**Discussion by BFergerson** [apache/skywalking#9007] [General]
It looks like SkyWalking is not detecting which endpoint logs are being produced by. For example, the following shows no logs: http://demo.skywalking.apache.org/dashboard/GENERAL/Endpoint/YWdlbnQ6OnNvbmdz.1/YWdlbnQ6OnNvbmdz.1_R0VUOi9zb25ncy90b3A=/General-Endpoint

This is despite the fact that the endpoint does produce logs: https://github.com/apache/skywalking-showcase/blob/ed83319ac7798c47ab1a8efb3d7e3187e37498cb/services/songs-service/src/main/java/org/apache/skywalking/showcase/services/song/controller/SongController.java#L39

---

It seems like a rather easy issue to solve. I'd just like to confirm it's a real issue and hear thoughts on where the fix should be implemented. My first thought was to have the agent check for an HTTP EntrySpan, but I think it might make more sense to implement on the OAP side.

**Comment by wu-sheng** [comment]
It is an issue, I knew for a while, but didn't fix yet.
Welcome to submit a pull request.

Agent seems not adding an endpoint into the log report.
